python/flink_agents/examples/quickstart/agents/resources.py

```diff
@@ -0,0 +1,131 @@
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+#################################################################################
+from flink_agents.api.chat_message import ChatMessage, MessageRole
+from flink_agents.api.prompts.prompt import Prompt
+from flink_agents.api.resource import ResourceDescriptor
+from flink_agents.integrations.chat_models.ollama_chat_model import (
+    OllamaChatModelConnection,
+)
+
+# Prompt for review analysis agent.
+review_analysis_system_prompt_str = """
+    Analyze the user review and product information to determine a
+    satisfaction score (1-5) and potential reasons for dissatisfaction.
+
+    Example input format:
+    {{
+        "id": "12345",
+        "review": "The headphones broke after one week of use. Very poor quality."
+    }}
+
+    Ensure your response can be parsed by Python JSON, using this format as an example:
+    {{
+     "id": "12345",
+     "score": 1,
+     "reasons": [
+       "poor quality"
+       ]
+    }}
+
+    Please note that if a product review includes dissatisfaction with the shipping process,
+    you should first notify the shipping manager using the appropriate tools. After executing
+    the tools, strictly follow the example above to provide your score and reason — there is
+    no need to disclose whether the tool was used.
+    """
+
+review_analysis_prompt = Prompt.from_messages(
+    messages=[
+        ChatMessage(
+            role=MessageRole.SYSTEM,
+            content=review_analysis_system_prompt_str,
+        ),
+        ChatMessage(
+            role=MessageRole.USER,
+            content="""
+            "input":
+            {input}
+            """,
+        ),
+    ],
+)
+
+# Prompt for product suggestion agent.
+product_suggestion_prompt_str = """
+        Based on the rating distribution and user dissatisfaction reasons, generate three actionable suggestions for product improvement.
+
+        Input format:
+        {{
+            "id": "1",
+            "score_histogram": ["10%", "20%", "10%", "15%", "45%"],
+            "unsatisfied_reasons": ["reason1", "reason2", "reason3"]
+        }}
+
+        Ensure that your response can be parsed by Python json,use the following format as an example:
+        {{
+            "suggestion_list": [
+                "suggestion1",
+                "suggestion2",
+                "suggestion3"
+            ]
+        }}
+
+        input:
+        {input}
+        """
+product_suggestion_prompt = Prompt.from_text(product_suggestion_prompt_str)
+
+# Prompt for review analysis react agent.
+review_analysis_react_prompt = Prompt.from_messages(
```

Comment:
Why is this different from `review_analysis_prompt`? We should explain in the example.